sshd/src/main/scala/ammonite/repl/sshd/ShellSession.scala

```diff
@@ -0,0 +1,63 @@
+package ammonite.repl.sshd
+
+import java.io.{InputStream, OutputStream}
+
+import ammonite.repl.sshd.util.Implicits
+import org.apache.sshd
+
+/**
+ * Implementation of ssh server's remote shell session, which will be serving remote user.
```

Comment:
This isn't a very useful doc-comment, and the `@param` description below is just C&P and serves no purpose. What I want to know: how does this relate to the `SshServer` and `SshRepl`? I suspect a new `ShellSession` is created every time someone SSHs in, but I'm not certain.